editor/nuxt.config.js

```diff
@@ -212,16 +212,16 @@ export default {
   ** See https://nuxtjs.org/api/configuration-build/
   */
   build: {
-    plugins: [
-      new MonacoWebpackPlugin({
-        features: [
-          '!goToDefinitionCommands',
-          '!goToDefinitionMouse',
-          '!referenceSearch'
-        ],
-        languages: ['css', 'html'],
-      })
-    ],
+    // plugins: [
```

Comment:
This doesn't make sense to strip, but can we work out why this is failing to build from a fresh clone?

Reply:
Not sure why MonacoWebpackPlugin isn't building... what environment are you building with?

Reply:
This one is on a Windows 10 PC. I put the nodejs version in the PR description too

Reply:
Looks like it needed node v16